src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtilsUnitTest.java

```diff
@@ -30,40 +34,223 @@
 
         final ArrayList<Object[]> testArgs = new ArrayList<>();
 
-        final ArrayList<Integer> baseArgs = new ArrayList<>();
-        baseArgs.add(DataSourceUtils.MIN_MAJOR_VERSION_NUMBER);
-        baseArgs.add(DataSourceUtils.MIN_MINOR_VERSION_NUMBER);
-        baseArgs.add(DataSourceUtils.MIN_YEAR_RELEASED);
-        baseArgs.add(DataSourceUtils.MIN_MONTH_RELEASED);
-        baseArgs.add(DataSourceUtils.MIN_DAY_RELEASED);
+        // =============================================================
+        // First do tests that check the major / minor version numbers:
+        // -------------------------------------------------------------
 
-        for ( int offset = -1 ; offset < 2; ++offset ) {
-            for ( int i = 0; i < baseArgs.size(); ++i ) {
+        final int MINOR_IT_MAX_VAL = 20;
 
-                final ArrayList<Object> argList = new ArrayList<>();
+        // major < MIN_MAJOR (minor shouldn't matter)
+        for (int minor = 0; minor < MINOR_IT_MAX_VAL; ++minor) {
+            testArgs.add(
+                    new Object[]{
+                            DataSourceUtils.MIN_MAJOR_VERSION_NUMBER - 1, minor, DataSourceUtils.MIN_DATE, false
+                    }
+            );
+        }
+
+        // major == MIN_MAJOR (minor does matter)
+        for (int minor = 0; minor < MINOR_IT_MAX_VAL; ++minor) {
+            testArgs.add(
+                    new Object[]{
+                            DataSourceUtils.MIN_MAJOR_VERSION_NUMBER, minor, DataSourceUtils.MIN_DATE,
+                            (minor == DataSourceUtils.MIN_MINOR_VERSION_NUMBER || minor == DataSourceUtils.MAX_MINOR_VERSION_NUMBER)
```

Comment:
I don't think this condition is quite right. If major == MIN_MAJOR, then minor must be >= MIN_MINOR. If major also happens to be == MAX_MAJOR (which it does when MIN_MAJOR == MAX_MAJOR), then minor also needs to be <= MAX_MINOR. So I think we want:

`minor >= DataSourceUtils.MIN_MINOR_VERSION_NUMBER && (major != DataSourceUtils.MAX_MAJOR_VERSION_NUMBER || minor <= DataSourceUtils.MAX_MINOR_VERSION_NUMBER)`

Reply:
Yeah - this was tailored to the current values of the MIN_ and MAX_ minor versions.

I'll update the conditions.